server/src/main/antlr3/org/apache/iotdb/db/sql/parse/TqlLexer.g

```diff
@@ -538,3 +538,8 @@ WS  :   ( ' '
         | '\n'
         ) {$channel=HIDDEN;}
     ;
+
+Duration
```

Comment:
Hi, it will be better to use "DURATION" to keep it consistent with other rules.

Reply:
OK, I'll fix it.